packages/user/src/components/UsersTable/AllUsersTable.tsx

```diff
@@ -162,13 +162,29 @@ export const AllUsersTable = ({
       accessorKey: "status",
       header: t("table.defaultColumns.status"),
       cell: ({ row: { original } }) => {
+        const getValue = () => {
+          if (original.isActiveUser) {
+            if (original.disabled) {
+              return t("status.disable");
+            } else {
+              return t("status.active");
+            }
+          } else {
+            return t("status.invited");
+          }
+        };
```

Comment:
Inverse if rules to simplify the logic and make it more legible.
```suggestion
        const getValue = () => {
          if (!original.isActiveUser) return t("status.invited");

          if (original.disabled) return t("status.disabled");

          return t("status.active");
};
```